README.md

```diff
@@ -27,7 +27,7 @@ Or, if you don't have an API Key:
 
 Then, you can create an API Key:
 
-    >>> apk = site.apikeys.create({"name": "My API Key"})
+    >>> apk = site.apikeys.create({"body": {"name": "API Key"}})
     >>> apk.refresh()
     >>> site = pycrunch.connect(api_key=apk.body["key"], site_url="https://your-domain.crunch.io/api/")
```

Comment:
Shouldn't 
```
site = pycrunch.connect(api_key=apk.body["key"]
```
be:

```
site = pycrunch.connect(api_key=apk.body["name"]
```

? 


Something else that would be helpful for is an example of how to connect to a host with a self-signed certificate.

Reply:
A small thing should you choose to include: 

in my actual terminal session, instead of the second `site` I named the variable `site_via_api_key`. This way, in case something goes wrong then I haven't destroyed my original connection.

Reply:
`apk.body["key"]` is what we want because `key` is the actual token whereas `name` is the name from the request payload:

```
{'key': 'apk-DFIJFIJWIEJIJFKSJLKKDJKFJSLLSLSL',
 'name': 'API Key',
 'created': '2022-01-27T21:48:00.684000+00:00',
 'last_used': None,
 'data': {'remote_ip': '127.0. 0.1'}}
 ```
 
 I actually don't have a way to bypass self-signed certs. The few times I need to point to my local environment I send the request directly to the python server instead of nginx.
 
 I'll update the example to include the use of `site_via_api_key`

Reply:
Thanks. Just verified that all of this indeed works. 

> I  actually don't have a way to bypass self-signed certs.

What munchkin does is set `verify=False` in the `requests` session.  But I think I'll just open this as a new issue for something in the future